cfgov/unprocessed/css/pages/event.less

```diff
@@ -88,6 +88,13 @@
     }
 }
 
+/* Event-Media-Container
+   ========================================================================== */
+
+.event-media_container {
+    border:1px solid @gray-10;
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/pull/1625/commits/36f6195becc4c28009ba7b84c31290ffe07277ae